packages/shared/src/components/squads/SquadPageHeader.tsx

```diff
@@ -84,6 +90,7 @@ export function SquadPageHeader({
       <div
         className={classNames(
           'absolute bottom-0 w-full translate-y-1/2 px-6 laptop:px-0 bg-theme-bg-primary',
+          tourIndex === TourScreenIndex.Post && 'highlight-pulse',
```

Comment:
Was it not easier to use the useTutorial for this?
It would need less drilldown right